js/client/fetcher.js

```diff
@@ -0,0 +1,57 @@
+const queries = {
+    "getParams": [
+        `{ dataquery(func: has(hashtags), first: 100, offset: ${getRandomInt(1000)}) { hashtags } }`,
```

Comment:
this way random value is calculated once and gets frozen for entire running time. Wouldn't you want to use new random offset every time?